docs/acb.md

```diff
@@ -154,4 +156,8 @@ The `build` property of a step specifies how to build a set of images. If build
 
 ### startDelay
 
-`startDelay` can be used to delay a step's execution. This is an integer value measured in seconds.
\ No newline at end of file
+`startDelay` can be used to delay a step's execution. This is an integer value measured in seconds.
+
+### ignoreErrors
+
+If `ignoreErrors` is set to `true`, the step will be marked as complete regardless of whether or not an error occurred during its execution.
```

Comment:
it's better to clarify the default value for optional settings.

Reply:
Updated.